Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public sealed class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float movingSpeed;
+    
+    private CharacterController _characterController;
+    private Vector2 _movementInput;
+    private Vector2 _moveDirection;

```

Comment:
Als ik kijk naar de code en hoe dit variable gebruikt wordt zou ik het zelf denk ik _moveVelocity noemen ipv _moveDirection.

_moveDirection suggereert dat het puur en alleen de richting waarop het object beweegt opslaat, maar volgens mij slaan jullie ook de snelheid en de daadwerkelijke beweging hierin op, in welk geval het een velocity wordt ipv alleen een direction.